parameter/store.go

```diff
@@ -42,8 +42,3 @@ type ObjectStore interface {
 	ReadObject(ctx context.Context, p string, out interface{}) error
 	WriteObject(ctx context.Context, p string, in interface{}) error
```

Comment:
Is the purpose of ReadObject and WriteObject to return values of a particular type?

Reply:
Yes of an object type